README.md

```diff
@@ -14,6 +14,22 @@ pip3 install -r requirements.txt
 export PYTHONPATH="$(pwd)"
 ```
 
+### Install pre-commit
+```
+brew install pre-commit
+or
+pip install pre-commit
+
+pre-commit install
+```
+
+If the check lint when you commit
```

Comment:
I don't understand what does it mean?

Reply:
Original I want to explain 
if the lint fail the result is ~
if the lint pass the result is ~
maybe I should add "###" in this line start?

Reply:
I updated the readme